code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -216,14 +216,10 @@
 	if(!can_sneak_attack)
 		return
 
-	var/staggerslow_stacks = 2 SECONDS
-	var/paralyzesecs = 1 SECONDS
-	var/flavour
-	if(owner.m_intent == MOVE_INTENT_RUN && ( owner.last_move_intent > (world.time - HUNTER_SNEAK_ATTACK_RUN_DELAY) ) ) //Allows us to slash while running... but only if we've been stationary for awhile
-		flavour = "vicious"
-	else
-		armor_mod += sneak_attack_armor_pen
```

Comment:
whole thing was seemingly handled in alien_attack stuff elsewhere while this is just for stun and whatever